### DISCUSSION TITLE
Support Copy/Paste ALL in Waveform View

### DISCUSSION THREAD

**Discussion by LeonCheung** [SubtitleEdit/subtitleedit#8250] [Ideas]
Simply as the title says. :)   Currently it seems we cannot do Ctrl C on any selected line(s) in Waveform view, so we have to go back to list view, copy the line(s), then go back to Waveform again and paste.

Further Thoughts
--
Some context menu items/shortcuts could also be considered, like:

`Copy`  (for any selected line(s))
`Copy/paste next to selection` (right after the selection, with min gap)
`Copy/paste to cursor position` (paste to video cursor)
`Copy/paste as overlap` (exactly overlap the selection, especially nice for ASSA)

**Comment by rRobis** [comment]
I noticed this too, that i can't copy paste waveforms, while, maintaining their length (if i copy from list view, it seems that it just pastes as text without timing?)
